Client/Frontend/Browser/BraveWebView.swift

```diff
@@ -46,4 +46,35 @@ class BraveWebView: WKWebView {
         lastHitPoint = point
         return super.hitTest(point, with: event)
     }
+
+    private func generateJavascriptFunctionString(functionName: String, args: [Any]) -> String {
+        var javascript = "\(functionName)("
+        for arg in args {
+            javascript += "'\(String(describing: arg).escapeHTML())', "
+        }
+
+        // Drop , and " "
+        javascript = javascript.dropLast(2) + ")"
+        return javascript
```

Comment:
Some Swift usage optimizations:

```suggestion
        let argsJS = args
          .map { String(describing: $0).escapeHTML() }
          .joined(separator: ", ")
        return "\(functionName)(\(argsJS))"
```